priv/templates/nimble_template/.github/wiki/_Sidebar.md

```diff
@@ -2,3 +2,8 @@
 
 - [[Home]]
 - [[Getting Started]]
+
+## Infrastructure
+
+- [[Application Status]]
+- [[Configuring Variables|Variable Configuration]]
```

Comment:
Should we consider using the same name? 🤔 💭  either updating the title or the filename.

Reply:
changed it in https://github.com/nimblehq/elixir-templates/pull/159/commits/06a24ca05d89961279d29d39e0542e117dafe7c4 @rosle